src/playground/src/components/playground/ChatParamsCard.tsx

```diff
@@ -98,7 +98,7 @@ export const ChatParamsCard = ({
         <ParamSelect
           label="Top P"
           explain="Similar to temperature, this controls randomness but uses a different method. Lowering Top P will narrow the model's token selection to likelier tokens. Increasing Top P will let the model choose from tokens with both high and low likelihood. Try adjusting temperature or Top P but not both."
-          onUpdate={(newValue) => tokenUpdate("topP", newValue)}
+          onUpdate={(newValue) => tokenUpdate("topP", Number(newValue))}
```

Comment:
Don't use the `Number` constructor as that creates an object of type `Number` rather than a number. If we need it to be numerical use `parseInt`